src/components/Folder.js

```diff
@@ -101,12 +102,19 @@ class Folder extends Component {
       </div>
     );
 
+    const folderText = `The folder allows you to save your graphs so you 
+                          can come back to them at a later time.`;
+
     return (
       <div
         className={classNames('Folder', {
           'Folder-expanded': expanded
         })}
       >
+        <div className="Component-header">
```

Comment:
`Folder-header` to follow the naming convention thus far. Repeat across other panes with their respective names.

Reply:
Done